cmd/body-based-routing/main.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package main is the entry point for the server.
+package main
+
+import (
+	"fmt"
+	"net"
+	"log"
+
+	"sigs.k8s.io/gateway-api-inference-extension/pkg/body-based-routing/service"
+	eppb "github.com/envoyproxy/go-control-plane/envoy/service/ext_proc/v3"
+	"google.golang.org/grpc"
+)
+
+func main() {
+	address := "0.0.0.0:8181"
```

Comment:
should change to `:8181` to support both ipv4 and ipv6

Reply:
Done